samples/nrf9160/lwm2m_client/sample_description.rst

```diff
@@ -277,25 +277,17 @@ Avoiding re-writing credentials to modem
 ----------------------------------------
 
 Every time the sample starts, it provisions the keys to the modem and this is only needed once.
-To speed up the start up, you can prevent the provisioning by completing the following steps:
+To speed up the start up, you can prevent the provisioning by completing the following steps using |VSC|:
```

Comment:
```suggestion
To speed up the start up, you can prevent the provisioning by completing the following steps using |nRFVSC|:
```

Reply:
I had checked our NCS documentation, and I believe, because we are talking about the tool in this scenario, it should be |VSC|.
@greg-fer : Please confirm.